lamindb/schema/_core.py

```diff
@@ -23,8 +22,45 @@ def draw(view=True):
 
 def list_entities():
     """Return all entities."""
-    metadata = sql.MetaData()
-    engine = settings.instance.db_engine()
-    metadata.reflect(bind=engine)
+    metadata = get_db_metadata()
     table_names = [table.name for table in metadata.sorted_tables]
     return table_names
+
+
+def get_db_metadata():
```

Comment:
It's completely fine to add this here, and you can go ahead with merging the PR!

However, I think this can be moved to `lndb_setup` in the next refactor. ☺️ I'll be happy to do that if you agree! No need to act on this right now.